src/task_8/task_8_2.py

```diff
@@ -0,0 +1,23 @@
+"""
+Даны три слова. Выяснить, является ли хоть одно из них палиндромом
+("перевертышем"), т. е. таким, которое читается одинаково слева направо и
+справа налево. (Определить функцию, позволяющую распознавать слова
+палиндромы.)
+"""
+
+def palindrom(x: list) -> list:
+    """
+    Функция по определению слов палиндромов.
+    На вход принимает произвольный список из слов.
+    На выходе возвращает новый список из слов палиндромов
+    :param x: list
+    :return: new list
+    """
+    palindrom_words = []
+    for i in x:
+        if i == i[::-1]:
+            palindrom_words.append(i)
+    return palindrom_words
```

Comment:
Попробуйте использовать генератор списков

```suggestion
    return [i for i in x if i == i[::-1]]
```